### PR TITLE
API obj terminology update - "Updating clusters"

### PR DESCRIPTION
**Pull request by maxwelldb**
Implementing [style changes](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#api-object-formatting) for all modules that are included in "Updating clusters" assemblies.

### CODE REVIEW

File: modules/images-configuration-registry-mirror.adoc [hunk 1]
```diff
@@ -209,5 +209,4 @@ troubleshoot the problem.
 * The main registry will only be used if no other mirror works.
 * From the system context, the `Insecure` flags are used as fallback.
 * The format of the `/etc/containers/registries` file has
-changed recently. It is now version 2 and in TOML format.
-*
+changed recently. It is now version 2 and in TOML format.
```
Comment (jboxman) [inline modules/images-configuration-registry-mirror.adoc:212]: What's TOML format?

**Review by jboxman — APPROVED** [review]
One question, but otherwise LGTM.

File: modules/images-configuration-registry-mirror.adoc (hunk 1 above)
Comment (maxwelldb) [reply modules/images-configuration-registry-mirror.adoc:212]: @jboxman https://toml.io/en/

**Pull request merged**